<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_Status">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>611</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Interface">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Dist">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_light">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance [cm]</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_Dist">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>511</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="alarmButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Alarm Checked</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clearText">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sens_1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ON/OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sens_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ON/OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sens_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ON/OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sens_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ON/OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sens_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ON/OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sens_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ON/OFF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
